src/sources/internal_logs.rs

```diff
@@ -64,26 +65,26 @@ async fn run(
         .chain(tokio_stream::wrappers::BroadcastStream::new(
             subscription.receiver,
         ))
+        .filter_map(|log| future::ready(log.ok()))
```

Comment:
Just curious, is it worth combining the `filter_map` and `filter` logic below? This seems more readable, but is there any overhead to consider?

Reply:
Considering the `filter` has been dropped, this question can be resolved.
It's probably something to bench. I don't really know what to expect considering that we would use some `and_then` and closures in the filter map...